otx/algorithms/detection/configs/detection/resnet50_dino/data_pipeline.py

```diff
@@ -0,0 +1,115 @@
+"""Data pipeline for DINO."""
+# dataset settings
+dataset_type = "CocoDataset"
+data_root = "data/coco/"
+img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+
+# train_pipeline, NOTE the img_scale and the Pad's size_divisor is different
+# from the default setting in mmdet.
+train_pipeline = [
+    dict(type="LoadImageFromFile"),
+    dict(type="LoadAnnotations", with_bbox=True),
+    dict(type="RandomFlip", flip_ratio=0.5),
+    dict(
+        type="AutoAugment",
+        policies=[
+            [
+                dict(
+                    type="Resize",
+                    img_scale=[
```

Comment:
Could you make the `_img_scale = [480,1333], ...` and reuse this variable to this line and line#48

Reply:
Fixed at 53359e9